### PR TITLE
Build for 4.5.2 RC1

### PR DESCRIPTION
**Pull request by sandrobonazzola**
## Changes introduced with this PR

Build for 4.5.2 RC1. Aligned with master changes.

## Are you the owner of the code you are sending in, or do you have permission of the owner?

y

### CODE REVIEW

File: ovirt-release-host-node.spec.in [hunk 1]
```diff
@@ -180,8 +180,7 @@ systemctl restart cockpit.service >/dev/null 2>&1
 
 PYTHON=$(command -v python3 || command -v python)
 
-pushd %{_sysconfdir}/yum.repos.d
-for REPO in $(find -maxdepth 1 -type f -name 'CentOS-oVirt-*.repo' -printf '%f\n');
```
Comment (sandrobonazzola) [inline ovirt-release-host-node.spec.in:184]: @lveyde not sure about this usage of find compared to master code, can you please review?

Comment (lveyde) [reply ovirt-release-host-node.spec.in:184]: It looks OK to me, but we'll of course will need to test it.

**Review by lveyde — APPROVED** [review]

File: ovirt-release-host-node.spec.in (hunk 1 above)
Comment (lveyde) [reply ovirt-release-host-node.spec.in:184]: I'm actually added the find code, that was removed by that patch, but it misses the search ovirt-*.repo files

**Review by lveyde — CHANGES_REQUESTED** [review]

File: ovirt-release-host-node.spec.in (hunk 1 above)
Comment (lveyde) [reply ovirt-release-host-node.spec.in:184]: Sent PR#247 to sync the code in master branch with the code in ovirt-4.5 branch.

**Comment by sandrobonazzola** [comment]
Abandoning in favor or @lveyde 's patches

**Pull request closed without merging**